pyproject.toml

```diff
@@ -27,7 +27,19 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-  "bioio-base>=0.2.0",
+  "bioio-base>=0.3.0",
+  "dask[array]>=2021.4.1",
+  "fsspec>=2022.8.0",
+  "imagecodecs>=2020.5.30",
+  "numpy>=1.16",
+  "lxml>=4.6,<5",
```

Comment:
`bioio` is already enforcing `ome-types>=0.4.0` so this should probably migrate to that too. Upon migrating `lxml` becomes available as an extra so we can install it through `ome-types` ([like is done in bioio](https://github.com/bioio-devs/bioio/blob/main/pyproject.toml#L36))

Reply:
resolved in [d07550a](https://github.com/bioio-devs/bioio-ome-tiff/pull/2/commits/d07550a7c2d5aac2de6a00a54ade1fc60281badd)